source/legacy.txt

```diff
@@ -37,6 +39,26 @@ In addition to updating your application to handle any necessary changes,
 always check for any other differences in options and return values before
 moving it to production.
 
+Current API Benefits
```

Comment:
There are other reasons that have nothing to do with server support, e.g. support for POJOs/records.  Should we also mention some of those?

Reply:
Info about POJOs and records has been added!